_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -80,11 +80,8 @@ test.describe('Popular Repositories', () => {
     });
 
     await test.step('Use kebab menu to delete a repo', async () => {
-      const filterInput = page.getByRole('textbox', { name: 'Filter by name/url' });
-      // We need to remember to clear the filter after specifying it above!
-      await filterInput.clear();
-      // Now we can enter a new value...
-      await filterInput.fill(repoName);
+      const searchInput = page.getByRole('textbox', { name: 'Filter by name/url' });
```

Comment:
ouf of interestes, how is this helping with `Fix test hang on popular repo delete ` :thinking: (adding commit message would be useful).

You technically removed the clearing of the input - was there issue with it?

Reply:
The clear took 20ms, so just re-filling was faster, which prevented an occasional test failure.